components/Button/button.module.scss

```diff
@@ -0,0 +1,3 @@
+.linkAsButton {
```

Comment:
if it's only one style file in the component folder, you can name this file as `index.module.scss`